core/src/main/java/org/apache/accumulo/core/fate/zookeeper/ZooCache.java

```diff
@@ -150,7 +153,7 @@ private class ZCacheWatcher implements Watcher {
     @Override
     public void process(WatchedEvent event) {
       if (log.isTraceEnabled()) {
-        log.trace("{}", event);
+        log.trace("{} {}", cacheId, event);
```

Comment:
Add colon for easier parsing 
```suggestion
        log.trace("{}: {}", cacheId, event);
```